packages/lib-classifier/src/components/Classifier/ClassifierContainer.spec.js

```diff
@@ -0,0 +1,715 @@
+      .persist()
+      .get('/field_guides')
+      .reply(200, { field_guides: [] })
+      .get('/tutorials')
+      .query(true)
+      .reply(200, { tutorials: [] })
+  }
+
+  class MockSubjectImage {
+    constructor () {
+      this.naturalHeight = 1000
+      this.naturalWidth = 500
+      setTimeout(() => {
+        this.onload()
+      }, 500)
+    }
+  }
+
+  function withGrommet(store) {
+    /*
+    The Classifier uses Grommet but isn't wrapped in the Grommet context,
+    so add that here.
+    Also reset the SWR cache between tests.
+    https://swr.vercel.app/docs/advanced/cache#reset-cache-between-test-cases
+    */
+    return function Wrapper({ children }) {
+      return (
+        <SWRConfig value={{ provider: () => new Map() }}>
+          <Grommet theme={zooTheme}>
+            {children}
+          </Grommet>
+        </SWRConfig>
+      )
+    }
+  }
+
+  describe('anonymous volunteers', function () {
+    let taskAnswers, subjectsRequest, workflowRequest
+    const subjectSnapshot = SubjectFactory.build({ locations: [{ 'image/png': 'https://foo.bar/example.png' }] })
+    const workflowSnapshot = branchingWorkflow
+    workflowSnapshot.strings = workflowStrings
+    const projectSnapshot = ProjectFactory.build({
+      links: {
+        active_workflows: [workflowSnapshot.id],
+        workflows: [workflowSnapshot.id]
+      }
+    })
+
+    beforeEach(async function () {
+      cleanStore()
+      sinon.replace(window, 'Image', MockSubjectImage)
+      const roles = []
+      nock('https://panoptes-staging.zooniverse.org/oauth')
+        .post('/token')
+        .reply(401,{ error: 'invalid_grant' })
+
+      mockPanoptesAPI()
+        .get('/project_preferences')
+        .query(true)
+        .reply(200, { project_preferences: [] })
+        .get('/project_roles')
+        .query(true)
+        .reply(200, { project_roles: [] })
+
+      subjectsRequest = nock('https://panoptes-staging.zooniverse.org/api')
+        .get('/subjects/queued')
+        .query(true)
+        .reply(200, { subjects: [subjectSnapshot, ...Factory.buildList('subject', 9)] })
+      workflowRequest = nock('https://panoptes-staging.zooniverse.org/api')
+        .get(`/workflows/${workflowSnapshot.id}`)
+        .query(true)
+        .reply(200, { workflows: [workflowSnapshot] })
+
+      const checkBearerToken = sinon.stub().callsFake(() => Promise.resolve(''))
+      const authClient = { ...defaultAuthClient, checkBearerToken }
+      render(
+        <ClassifierContainer
+          authClient={authClient}
+          project={projectSnapshot}
+          workflowID={workflowSnapshot.id}
+        />,
+        {
+          wrapper: withGrommet()
+        }
+      )
+      const taskTab = await screen.findByRole('tab', { name: 'TaskArea.task'})
+      const tutorialTab = screen.queryByRole('tab', { name: 'TaskArea.tutorial'})
+      const tabPanel = screen.queryByRole('tabpanel', { name: '1 Tab Contents'})
```

Comment:
```suggestion
      const tabPanel = await screen.findByRole('tabpanel', { name: '1 Tab Contents'})
```

Maybe change this to wait for the first task to be ready?